lib/factory_girl.rb

```diff
@@ -59,12 +59,14 @@ def self.reset_configuration
   # Parameters:
   # factories - which factories to lint; omit for all factories
   # options:
-  #   traits : true - to lint traits as well as factories
+  #   traits: true - to lint traits as well as factories
+  #   strategy: :create - to specify the strategy for linting
   def self.lint(*args)
     options = args.extract_options!
     factories_to_lint = args[0] || FactoryGirl.factories
-    strategy = options[:traits] ? :factory_and_traits : :factory
-    Linter.new(factories_to_lint, strategy).lint!
+    linting_strategy = options[:traits] ? :factory_and_traits : :factory
```

Comment:
Both this new variable and the previous `strategy` are somewhat confusing because it makes be think of the ["build strategy"](http://www.rubydoc.info/gems/factory_girl/file/GETTING_STARTED.md#Using_factories).

Would `linting_scope` or `linting_approach` be good alternative names?